sdk/containerregistry/container-registry/.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I think we've stopped comitting these :-/

Reply:
We still have some for lib authors who want to debug current file.